kafka/src/test/java/org/apache/apex/malhar/kafka/KafkaInputOperatorTest.java

```diff
@@ -275,10 +324,21 @@ public void testInputOperator(boolean hasFailure, boolean idempotent) throws Exc
     final LocalMode.Controller lc = lma.getController();
     lc.setHeartbeatMonitoringEnabled(false);
 
-    lc.runAsync();
```

Comment:
I guess I don't understand what that thread is doing that the main thread cannot achieve?